upstairs/src/lib.rs

```diff
@@ -817,6 +842,78 @@ impl Work {
     }
 }
 
+/// Implement XTS encryption
+/// See: https://en.wikipedia.org/wiki/Disk_encryption_theory#XEX-based_tweaked-codebook_mode_with_ciphertext_stealing_(XTS)
+pub struct UpstairsEncryptionContext {
```

Comment:
Nit: could we call this `EncryptionContext`?  There is no downstairs encryption context, and it's less to type :)

Reply:
Changed